src/entrypoint.sh

```diff
@@ -1,20 +1,90 @@
 #!/bin/bash
 
-if [ ! -f yarn.lock ]; then
-    echo "Can't find yarn.lock. Generate lock file and try again."
+if [ "$1" = "--help" ]; then
+    cat << EOF
+Usage: entrypoint supports the following arguments [ARGS]
+Arguments:
+  --check
+      In addition to generate dependencies info, check if all of them suttisfy
+      License requirements, fail if no
+  --help
+      Print this message
+EOF
+    exit 0
+fi
+
+CHECK=""
+if [ "$1" = "--check" ]; then
+    CHECK="$1";
+fi
+
+WORKSPACE=$(pwd)
+PROJECT=$WORKSPACE/project
+
+if [ ! -d $PROJECT ]; then
+    echo
+    echo "Error: The project directory is not mounted."
+    exit 1
+fi
+
+if [ ! -f $PROJECT/package.json ]; then
+    echo "Error: Can't find package.json file in the project directory. Commit it and then try again."
+    exit 1
+fi
+
+if [ ! -f $PROJECT/yarn.lock ]; then
+    echo "Error: Can't find yarn.lock file. Generate and commit the lock file and then try again."
+    exit 1
+fi
+
+if [ -n "$CHECK" ] && [ ! -f $PROJECT/DEPENDENCIES ]; then
+    echo "Error: Can't find DEPENDENCIES file. Generate and commit this file for the project using 'dash-license' and then try again."
     exit 1
 fi
 
-DASH_LICENSES=dash-licenses/core/target/org.eclipse.dash.licenses-0.0.1-SNAPSHOT.jar
+DASH_LICENSES_DIR=$WORKSPACE/dash-licenses
+DASH_LICENSES=$WORKSPACE/dash-licenses.jar
 if [ ! -f $DASH_LICENSES ]; then
-    echo "Can't find org.eclipse.dash.licenses-0.0.1-SNAPSHOT.jar. Rebuild 'nodejs-license-tool' image and try again."
+    echo "Error: Can't find dash-licenses.jar. Rebuild 'nodejs-license-tool' image and try again."
```

Comment:
From users perspective, they are not supposed to build container.
```suggestion
    echo "Error: Can't find dash-licenses.jar. Contact https://github.com/che-incubator/dash-licenses maintainers to fix the issue."
```

Reply:
fixed